docker-compose.yml

```diff
@@ -1,5 +1,23 @@
 version: '3.5'
 services:
+  nhsd.buyingcatalogue.identity.api.user.db:
+    image: ${REGISTRY:-nhsd}/buying-catalogue/identity-api-user-db:${TAG:-latest}
+    build: 
+      context: src/NHSD.BuyingCatalogue.Identity.UserDatabase
+    container_name: identity_api_user_db
+    volumes:
+      - nhsd_identity_user_db_volume:/var/opt/mssql/
+    networks:
+      - nhsd_bcnetwork
+    environment:
+      - ACCEPT_EULA=Y
+      - SA_PASSWORD=MyTestPassword123!

```

Comment:
My initial impulse is that this being different to the value in the sqlproj is going to make me sad somewhere down the line

Reply:
That's the `sa` password. The test password for the `NHSD` user is the same as in the sqlproj.